modules/installation-gcp-operator-permissions.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+// * installing/installing_gcp/installing-gcp-user-infra.adoc
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+// * installing/installing_gcp/installing-restricted-networks-gcp.adoc
+
+[id="installation-gcp-operator-permissions_{context}"]
+= GCP permissions required for {product-title} Operators
+
+When installing an {product-title} cluster on GCP, you must grant the appropriate GCP permissions based on the cluster Operators your installation requires to run.
+
+.Cloud Credentials Operator - Mint Mode
+
+* `resourcemanager.projects.get`
+* `serviceusage.services.list`
+* `iam.serviceAccountKeys.create`
+* `iam.serviceAccountKeys.delete`
+* `iam.serviceAccounts.create`
+* `iam.serviceAccounts.delete`
+* `iam.serviceAccounts.get`
+* `iam.roles.get`
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+
+.Cloud Credentials Operator - Passthrough Mode
```

Comment:
Hi Jianli, 

According to @akhil-rane I should not be adding a definitive list. Instead, I've opted to add a note with a link to the 'Passthorugh Mode' docs. The reason for this note is because they change with every OCP release. I did this at Akhil's advice.